<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgChartColor</class>
 <widget class="QDialog" name="DlgChartColor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>156</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chart color schema</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="1">
    <widget class="QCheckBox" name="chShowRoomDescription">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Reserved color</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Checkin color</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>95</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnSave">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnCancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>95</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="EColorLineEdit" name="leReserveColor"/>
   </item>
   <item row="1" column="1">
    <widget class="EColorLineEdit" name="leCheckinColor"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Show room description</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Do not disturbe color</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="EColorLineEdit" name="leDoNotDisturbeColor"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EColorLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">ecolorlineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>leReserveColor</tabstop>
  <tabstop>leCheckinColor</tabstop>
  <tabstop>leDoNotDisturbeColor</tabstop>
  <tabstop>chShowRoomDescription</tabstop>
  <tabstop>btnSave</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
